destination.go

```diff
@@ -69,24 +51,119 @@ func (d *Destination) Configure(ctx context.Context, cfg config.Config) error {
 	return nil
 }
 
-func (d *Destination) Open(_ context.Context) error {
-	// Open is called after Configure to signal the plugin it can prepare to
-	// start writing records. If needed, the plugin should open connections in
-	// this function.
+func (d *Destination) Open(_ context.Context) (err error) {
+	d.db, err = sqlx.Open("mysql", d.config.URL)
+	if err != nil {
+		return fmt.Errorf("failed to connect to mysql: %w", err)
+	}
+
 	return nil
 }
 
-func (d *Destination) Write(_ context.Context, _ []opencdc.Record) (int, error) {
-	// Write writes len(r) records from r to the destination right away without
-	// caching. It should return the number of records written from r
-	// (0 <= n <= len(r)) and any error encountered that caused the write to
-	// stop early. Write must return a non-nil error if it returns n < len(r).
-	return 0, nil
+func (d *Destination) Write(ctx context.Context, recs []opencdc.Record) (int, error) {
```

Comment:
It would be good to support batching in this connector too... It's not a must-have for this PR, but if there isn't a strong against that, I suggest creating an issue for that.

Reply:
Yup, opened [here](https://github.com/conduitio-labs/conduit-connector-mysql/issues/63).

Reply:
Lovely, thanks!